pkg/controller/windowsmachine/windows/windows.go

```diff
@@ -146,6 +148,15 @@ func (vm *windows) CopyFile(filePath, remoteDir string) error {
 	return nil
 }
 
+func (vm *windows) FileExists(path string) (bool, error) {
+	out, err := vm.Run("Test-Path "+path, true)
+	if err != nil {
+		return false, errors.Wrapf(err, "error checking if file %s exists on Windows VM %s. output: %s",
+			path, vm.ID(), out)
```

Comment:
No need to return `out` in the error. I will be logging that error in #147 and that should suffice.